public/javascripts/previewers/viewer_hop/package.json

```diff
@@ -0,0 +1,6 @@
+{ 
+  "name": "VHOP",
+  "main": "viewer_hop.js",
+  "file": true,
+  "contentType": ["model/ply", "application/octet-stream"]
```

Comment:
To avoid using `application/octet-stream` and having the previewer trigger on any file not recognized we can add
the following to `conf/mimetypes.conf`:
```
mimetype.nxz=model/nxz
mimetype.NXZ=model/nxz
```
This line can then be changed to "contentType": ["model/ply", "model.nxz"].

Files that end up in `.ply` are recognized as  `model/ply` because of these entries in the same file:

```
mimetype.ply=model/ply
mimetype.PLY=model/ply
```

Reply:
Hi @lmarini, good point. I changed the `package.json` file as requested.

Reply:
Thank you @cconsta1 Can you also add the nxz entries to the `conf/mimetypes.conf` folder?

Oh and I forgot, can you also add an entry to the changelog? Thanks!

Reply:
Hi @lmarini. Done! :)

Reply:
Thank you! Will try to include in v1.12.0 release.

Reply:
Thanks @lmarini!